src/CodeStyle/Tools/CodeStyleConfigFileGenerator.csproj

```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework Condition="'$(DotNetBuildFromSource)' == 'true'">net7.0</TargetFramework>
```

Comment:
After some discussion with others, we'd prefer `net7.0` be added into the existing `<TargetFramework>` vs. having a new one which is conditioned on `$(DotNetBuildFromSource)`. We still have our concerns about the testing burden increase this creates but we will deal with that in a follow up PR. 

Adding the comment here but applies to all instances of this change.

Reply:
If we target net6.0 at all, then we will need 6.0 targeting packs which are not available in source build. Would something like this be acceptable?

```
    <TargetFramework>net6.0;net7.0</TargetFramework>
    <TargetFramework Condition="'$(DotNetBuildFromSource)' == 'true'">net7.0</TargetFramework>
```

Reply:
No. We've historically found that `Condition` paired with `TargetFramework` to be unmaintainable.

Reply:
I'm sorry but `net6.0` is a reality that we cannot escape. We've also found that conditioned `TargetFramweork` is a maintainenance nightmare for us. 

I'm still perplexed at why we're being forced to ship `net7.0` in our TF now. It's never been required before and analyzers / generators have been around since 1.0